rustls-cert-gen/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "rustls-cert-gen"
+version = "0.1.0"
+license.workspace = true
+edition.workspace = true
+
+[[bin]]
```

Comment:
The `[[bin]]` section isn't needed -- Cargo should pick it up and do the right thing implicitly.